src/components/CardListItem.js

```diff
@@ -62,10 +62,9 @@ const styles = StyleSheet.create({
 /* this works properly, we do not want that warning */
 const stylesWithProps = ({ horizontal, orientation, dimensions }) => {
   // image width should be only 70% when rendering horizontal cards, otherwise substract paddings
-
   const width = horizontal
     ? imageWidth() * 0.7
-    : orientation === 'landscape'
+    : orientation === 'landscape' || (orientation === 'portrait' && dimensions.width > 450)
```

Comment:
we do not need to check for `orientation === 'portrait'`, because we do not care. if the device is greater than 450 in width, than we want to apply the wrapper.